dvach-browser/VLCPlayerViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  VLCPlayerViewController.swift
+//  dvach-browser
+//
+//  Created by Victor Cebanu on 2/3/20.
+//  Copyright © 2020 8of. All rights reserved.
+//
+
+import Foundation
+
+final class VLCPlayerViewController: UIViewController, VLCPlayerViewInput {
+    var interactor: VLCPlayerViewOutput!
+    var player: VLCMediaPlayer?
+
+    @IBOutlet weak var playerView: UIView!
+    @IBOutlet weak var progressBar: UISlider!
+    @IBOutlet weak var currentTimeLabel: UILabel!
+    @IBOutlet weak var timeLeftLabel: UILabel!
+    @IBOutlet weak var playButton: UIButton!
+    @IBOutlet weak var muteButton: UIButton!
+
+    private let defaultVideoJumpInterval: Int32 = 10 // TODO: Read from settings
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupPlayer()
+        interactor.viewDidLoad()
+    }
+
+    // IBAction
+
+    @IBAction func progressBarWasMoved(_ sender: UISlider) {
```

Comment:
We can move every private method to a private local extension. Code will be more organized, robust and predictable.